### DISCUSSION TITLE
Define another client

### DISCUSSION THREAD

**Discussion by fvga7** [nuxt-community/apollo-module#505] [Q&A]
Hi folks, 

i would like to ask you for help regarding my query where i need to use different client : 

`const { data: articleData } = await useAsyncQuery(ARTICLE_QUERY, { id, ...{clientId: 'admin'} });`

i cannot figure out whats wrong with it, as for example i am able to use the different client here in a mutation : 

```
const updateAnswer = async (answerId, votes) => {
  const { mutate } = useMutation(VOTE_MUTATION, {
    variables: { answerId, votes: votes + 1 },
    clientId: 'admin'
  });
  const result = await mutate();
  return result.data.updatePollAnswer;
};
```

this Mutation works, but the AsyncQuery not, could you please tell me whats wrong ?

**Comment by jumptrnr** [comment]
figure out?

**Comment by fvga7** [comment]
nope, i used different way to achieve my goal.
